Dockerfile

```diff
@@ -68,7 +68,6 @@ RUN apt-get update && \
         cvs \
         git \
         mercurial \
-        subversion \
```

Comment:
That change belongs to a previous commit.

Reply:
Fixed.